seqr/views/utils/pedigree_info_utils.py

```diff
@@ -323,11 +324,20 @@ def validate_fam_file_records(project, records, fail_on_warnings=False, errors=N
             if invalid_features:
                 errors.append(f'{individual_id} has invalid HPO terms: {", ".join(sorted(invalid_features))}')
 
+        affected_status_by_family[family_id].append(r.get(JsonConstants.AFFECTED_COLUMN))
+
     errors += [
         f'{individual_id} is included as {count} separate records, but must be unique within the project'
         for individual_id, count in individual_id_counts.items() if count > 1
     ]
 
+    no_affected_families = [
```

Comment:
This logic is duplicated inside of `_upload_data_loading_files` [here](https://github.com/broadinstitute/seqr/pull/4495/files#diff-d97edf947abfdd9ae291360399589f1b801ad8ca0586ea531ca2f60cf04927ceR159), can it be shared?